tests/test_dt.py

```diff
@@ -176,6 +176,20 @@ def test_coverage():
         core.test_coverage()
 
 
+def test_multiprocessing_threadpool():
+    # Verify that threads work properly after forking (#1758)
+    import multiprocessing as mp
+    from datatable.internal import get_thread_ids
+    parent_threads = get_thread_ids()
+    n = 4
+    pool = mp.Pool(processes=n)
+    child_threads = pool.starmap(get_thread_ids, [()] * n, chunksize=1)
+    assert len(child_threads) == n
+    for chthreads in child_threads:
+        assert len(parent_threads) == len(chthreads)
+        assert chthreads != parent_threads
```

Comment:
When `get_thread_ids()` is called, does it always return threads in the same order? If by accident the order is different between child and parent, but threads are the same, may be it is safer to do `assert Counter(chthreads) == Counter(parent_threads)`?

Reply:
Thread ids are always returned in the same order, in the sense that repeated calls to `get_thread_ids()` will always produce the same list. However, thread ids in child processes will necessarily be different from the thread ids of the parent, because children do not inherit threads from the parent. Therefore I test `chthreads != parent_threads` (it is allowed, though very unlikely, for some thread ids to accidentally be the same in 2 processes, but for all of them to be same is practically impossible). 
The standard does guarantee that all thread ids in a single process are unique, so there is no point in using `Counter`.

Reply:
I see, if they always come in the same order then it's ok.